rollup.config.js

```diff
@@ -249,4 +252,46 @@ export default {
       ],
     }),
   ],
-};
+  },{
+    input: path.resolve(path.join('Sources', 'vtk.js')),
+    output: {
+      file: path.join(outputDir, 'vtk.js'),
+      format: 'es',
+      hoistTransitiveImports: false,
+      intro() {
+        return `/* ${fs.readFileSync(
```

Comment:
Do we want to preserve the license in downstream apps that build using vtk.js? This is typically accomplished via `/*!` or a line with `@license`.

Reply:
Needing to preserve will complicate how we get it to be preserved...